src/layouts/Media/components/MediaDirectoryCard.tsx

```diff
@@ -45,27 +45,20 @@ export const MediaDirectoryCard = ({
         <ContextMenu.Button pos="absolute" />
         <ContextMenu.List>
           <ContextMenu.Item
-            icon={<BiEditAlt />}
-            as={RouterLink}
-            to={`/sites/${siteName}/media/${mediaType}/mediaDirectory/${encodedDirectoryPath}`}
-          >
-            <Text>Edit folder</Text>
-          </ContextMenu.Item>
```

Comment:
not sure if this is covered later on but how are we allowing them to edit the folder in this case?

Reply:
Wait current understanding is Edit folder just allows user to rename folder, pls correct me if otherwise

Reply:
1. "Edit ____" for folder / album / category: used to be the same as "open"
2. "Edit ____" for page / image / file: used to be the same as "rename" (= also called "Settings" on the modal)

Imo #1 is okay to get rid of becos it's the same as just clicking on the tile
#2 is covered by the new "Rename" option